packages/contrast-colors/index.js

```diff
@@ -279,6 +314,116 @@ function contrast(color, base) {
   }
 }
 
+function minPositive(r) {
+  if (!r) { throw new Error('Array undefined');}
```

Comment:
Could use `!Array.isArray(r)` here and give the message "Passed object is not an array."